LV_analysis/0_process_Pa_KEGG_pathways.py

```diff
@@ -0,0 +1,86 @@
+# ---
```

Comment:
Since I'm now using the pre-commit I have some duplicated files. I plan to refactor this repo after all the analyses are complete to either move all the .py files to a new directory or remove the nbconverted dir. I haven't decided just yet so I'll leave this for that future refactor PR.